spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -1092,6 +1095,10 @@ def fixture_record(fixture_name, indexer: @indexer)
         lc_facet = @record_no_call_number['lc_facet']
         expect(lc_facet).to be_nil
       end
+
+      it 'handles cases where the call number is invalid' do
```

Comment:
Could we make this description a little more specific?
```suggestion
      it 'does not index data into the lc_facet field if the call number is invalid' do
```